src/Helpers/babylon.videoDome.ts

```diff
@@ -75,6 +75,7 @@ module BABYLON {
             // configure material
             texture.coordinatesMode = BABYLON.Texture.FIXED_EQUIRECTANGULAR_MIRRORED_MODE; // matches orientation
             texture.wrapV = Texture.CLAMP_ADDRESSMODE; // always clamp the up/down
+            texture.video.crossOrigin = "anonymous";

```

Comment:
Please use texture.video.crossOrigin =Tools.CorsBehavior
Also I would prefer to update the videoTexture object directly

Reply:
I'll make the first change but am unsure what you mean by the second.
"Also I would prefer to update the videoTexture object directly"